web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/GetSickLeaveCertificatesImpl.java

```diff
@@ -60,6 +73,10 @@ public List<SjukfallEnhet> get(String careProviderId, List<String> unitIds, List
 
```

Comment:
Add sickLeaveLogMessageFactory.message(...) to log time to findAllSjukfallCertificate. Maybe call the log: GET_SICK_LEAVE_CERTIFICATES

Reply:
Sorry I didn't see this comment before implementing a new message type. Is this OK?